.github/workflows/haskell.yml

```diff
@@ -20,7 +20,7 @@ jobs:
     - uses: haskell/actions/setup@v1
       with:
         ghc-version: ${{ matrix.ghc }}
-        cabal-version: '3.4'
+        cabal-version: 'latest'
```

Comment:
not always last cabal is good, would `3.6` be too much careful?

Reply:
afaik 3.6 will pick 3.6.2.0, 3.6.0.0 suffered some bugs

Reply:
Fair enough.